unittests/Expression/SwiftParserTest.cpp

```diff
@@ -0,0 +1,69 @@
+//===-- SwiftParserTest.cpp --------------------------------------*- C++-*-===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+#include "Plugins/ExpressionParser/Swift/SwiftHost.h"
+#include "TestingSupport/TestUtilities.h"
+#include "lldb/Host/FileSystem.h"
+#include "lldb/Host/HostInfo.h"
+#include "lldb/Utility/FileSpec.h"
+#include "lldb/lldb-defines.h"
+#include "gtest/gtest.h"
+
+using namespace lldb_private;
+
+namespace {
+struct SwiftHostTest : public testing::Test {
+  static void SetUpTestCase() {
+    FileSystem::Initialize();
+    HostInfo::Initialize();
+  }
+  static void TearDownTestCase() {
+    HostInfo::Terminate();
+    FileSystem::Terminate();
+  }
+};
+} // namespace
+
+static std::string
```

Comment:
There is already another unit test (host?) that tests Resource directory paths. Could we put this there?

Reply:
You mean the `ClangParserTest` one? They are different resource directory and have different settings. It's better to have them separate